<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinTables</class>
 <widget class="QWidget" name="JoinTables">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page0">
      <layout class="QVBoxLayout" name="page0VerticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="Widgets::TitleCommentWidget" name="title_comment" native="true"/>
       </item>
       <item>
        <widget class="Widgets::SelectedTables" name="selected" native="true"/>
       </item>
       <item>
        <widget class="QLabel" name="warningLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>For merge tables you should select minimum 2 tables. Close the window and make right choice</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page1">
      <layout class="QVBoxLayout" name="page1VerticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="Widgets::EditGroupWidget" name="edit_group" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widgets::TitleCommentWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/title_comment_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Widgets::SelectedTables</class>
   <extends>QWidget</extends>
   <header>ui/widgets/selected_tables.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Widgets::EditGroupWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/edit_group_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
